src/main/java/com/knu/buga1chuk/util/FileUtils.java

```diff
@@ -0,0 +1,29 @@
+package com.knu.buga1chuk.util;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+public class FileUtils {
```

Comment:
The utility classes should always be final

Reply:
fix